content/ending-frozen-applications.md

```diff
@@ -0,0 +1,77 @@
+---
+title: Closing Frozen Applications and Desktops
+description: >
+  Linux provides several methods to kill frozen or hung up processes and applications using GUI applications or terminal commands.
+keywords:
+  - frozen applications
+  - frozen desktop
+  - frozen terminal
+  - hung up
+  - not responding
+  - System76
+
+facebookImage: /_social/article
+twitterImage: /_social/article
+
+hidden: false
+section:
+---
+
+## Closing Frozen Processes, Applications, and Desktops
+
+Linux offers many methods to stop hanging or frozen processes. The System Monitor application provides a GUI with features similar to the Windows Task Manager, while the `kill` command can end single or multiple processes from the Terminal.
```

Comment:
The new `xkill` section looks great-- should it be included in the summary up here?